iopipe/iopipe.py

```diff
@@ -223,10 +230,7 @@ def send(self, context=None):
         self.report['time_nanosec'] = \
             int((time.time() - self._time_start) * 1000000000)
 
-        # Falsify function_id
-        self.report['function_id'] = '0xDEADBEEF'
```

Comment:
You're dropping this function_id bit? do we use it at all/will that cause problems?